src/products/mozilla_org/manage_mozillaorg.md

```diff
@@ -0,0 +1,95 @@
+# Managing Mozilla.org
+
+Since the migration from .lang to Fluent, the L10n PM’s job is a lot simpler. The PM’s primary focus is on reviewing the string extractions provided by the bedrock team, provides feedback, manages the project through Pontoon and communicates important information to the communities.
```

Comment:
I don't think we need to tell in documentation that this is easier than the past. These documents are supposed to be stand-alone pieces of information for whoever needs to take over your work, so they need as much information as possible on the current status.

The second part of the sentence is disconnected ("The PM?s focus is on reviewing…, provides…, manages…").

nit: bedrock -> Bedrock.